ObjectOrientedProgramming/Memory Ownership/Placement new/src/task.cpp

```diff
@@ -0,0 +1,31 @@
+#include <new>
+
+class MyObject {
```

Comment:
Please add some fields to the object and initialize them in constructor. 
Also, would be nice to have more "realistic" example of an object, instead of just `MyObject`. For example, you give `Laptop` and `Student` classes as examples in another task, so perhaps something in a similar vein.